src/components/utils/src/threads/thread_posix.cc

```diff
@@ -100,11 +102,14 @@ void* Thread::threadFunc(void* arg) {
   pthread_cleanup_push(&cleanup, thread);
 
   thread->state_lock_.Acquire();
+  thread->thread_created_ = true;
```

Comment:
@pestOO, fixed in [ca04ae0](https://github.com/smartdevicelink/sdl_core/pull/707/commits/ca04ae01ea2e23d26e87ceb337e38dbe7bb04f1a) and [6a6fa50](https://github.com/smartdevicelink/sdl_core/pull/707/commits/6a6fa50f4c909cb87b2be555b462c1ae7a660cb1) (Qt/Win)